### DISCUSSION TITLE
Broadcasting to all clients

### DISCUSSION THREAD

**Discussion by car1ot** [uNetworking/uWebSockets.js#243] [Q&A] [answered]
How to broadcast data to all connected clients? For example, broadcasting every minute bot notifications etc.

**Comment by car1ot** [comment]
To broadcast, I should store all connected users in some variable? Or I could access to all connected users other way inside uws.App()?

**Comment by ghost** [comment]
Subscribe to a topic then publish to it. Check documentation

**Comment by car1ot** [comment]
> 
> 
> Subscribe to a topic then publish to it. Check documentation

??? Publish data to topic only allowed from one of active websockets.

**Comment by yovanoc** [comment, marked as answer]
You can do an `app.publish`

**Comment by car1ot** [comment]
Thanks

**Comment by ghost** [comment]
Which is written in the docs, you should read it
